chart/templates/database/db-statefulset.yaml

```diff
@@ -48,22 +38,12 @@ spec:
       replicas: {{ $replicas }}
       selector:
         matchLabels:
-          app: database
-          app.kubernetes.io/instance: {{ .Release.Name | quote }}
-          app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
-          app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
-          app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
-          helm.sh/chart: {{ include "kubecf.chart" . }}
+          {{- list . "database" | include "component.labels" | nindent 10 }}
```

Comment:
This should be okay as the old selector included the version, which would not have worked across upgrades; therefore modifying the statefulset selector must be okay.